### DISCUSSION TITLE
Same seests from different modules fetched only once

### DISCUSSION THREAD

**Discussion by zhanghan3** [webpack/webpack#18631] [Q&A]
Hi team!  I am using module federation feature but I am encountering an issue about assets import.
Let me simply my question because my project is really huge.

I have two modules, moduleA and moduleB. And they both import another package in my project, let's name it with `package1`.
When my website initialize, the two modules are both loaded. If moduleA loaded the `package1` first, the moduleB would not load `package1`, it will just use what moduleA loaded.

My question is: this behavior is Webpack feature or it is a bug.

What I expect is the two modules use their own imported `package` seperately because sometimes their code are little different.

**Comment by alexander-akait** [comment]
Can you create a small reproducible repo?
